### ISSUE TITLE
Activait UI needs update to reflect more options for loss functions

### ISSUE THREAD

**Issue by gurhar1133** [radiasoft/sirepo#4918]
We currently have a list of loss functions the user can select, but some loss functions have important input parameters that change their meaning.
eg.

```python
model.compile(
    ...
    loss='sparse_categorical_crossentropy',
   ...
)
```

```python
model.compile(
    ...
    loss=keras.losses.SparseCategoricalCrossentropy(from_logits=True),
   ...
)
```

The UI needs to give the user the ability to specify inputs to the loss function selection, and the template code needs to be able to generate the loss function as specified by the user in the UI
